backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -0,0 +1,35 @@
+[id="about-dr"]
+= About disaster recovery
+include::modules/common-attributes.adoc[]
+:context: about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from several disaster situations that might occur with their
+{product-title} cluster. As an administrator, you might need to follow one or
+more of the following procedures in order to return your cluster to a working
+state.
+
+xref:../../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[Recovering from lost master hosts]::
+This solution handles situations where you have lost the majority of your master
+hosts, leading to etcd quorum loss and the cluster going offline. As long as you
+have taken an etcd backup and have at least one remaining healthy master host,
+you can follow this procedure to recover your cluster.
++
+If applicable, you might also need to xref:../../backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-recovering-expired-certs[recover from expired control plane certificates].
+
+xref:../../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[Restoring back to a previous cluster state]::
+This solution handles situations where you want to restore your clsuter back to
+a previous state, for example, after a failed upgrade or if an administrator
+deletes something critical. As long as you have taken an etcd backup, you can
+follow this procedure to restore your cluster to a previous state.
++
+If applicable, you might also need to xref:../../backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-recovering-expired-certs[recover from expired control plane certificates].
+
+xref:../../backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-recovering-expired-certs[Recovering from expired control plane certificates]::
+This solution handles situations where your control plane certificates have
```

Comment:
@tnozicka Can you review this description for recovering from expired certs to make sure that it's accurate?

Preview: https://bz-1744569--ocpdocs.netlify.com/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html